web/modules/custom/joinup_federation/src/Plugin/Adms2ConvertPass/Pass41.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\joinup_federation\Plugin\Adms2ConvertPass;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\joinup_federation\JoinupFederationAdms2ConvertPassPluginBase;
+use Drupal\Tests\joinup_federation\Kernel\ConvertToAdms2StepTest;
+
+/**
+ * Conversion Pass #41.
+ *
+ * URI: dct:issued
+ * Type: N.A.
+ * Action: Updated
+ * Description:
+ * - Updated: Cardinality: 0..n -> 0..1.
+ *
+ * @Adms2ConvertPass(
+ *   id = "pass_41",
+ * )
+ */
+class Pass41 extends JoinupFederationAdms2ConvertPassPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function convert(array $data): void {
+    $results = $this->getTriplesFromGraph($data['sink_graph'],
+      NULL,
+      'http://www.w3.org/ns/dcat#issued'
+    );
+
+    $entities = [];
+    // In order to be fair, and since the notion of deltas does not exist in
+    // SPARQL, we are only going to keep the first instance found for the given
+    // value.
+    foreach ($results as $result) {
+      if (isset($entities[$result['subject']])) {
+        continue;
+      }
+      $entities[$result['subject']] = $result['object'];
+    }
+
+    foreach ($entities as $subject => $object) {
+      $query = <<<QUERY
+WITH <{$data['sink_graph']}>
+DELETE { <$subject> <http://www.w3.org/ns/dcat#issued> ?issued }
```

Comment:
For the issued it is probably fine to delete, for others it is probably safer to rely on the validator.